src/content/team/pauline_hans.md

```diff
@@ -1,5 +1,6 @@
 ---
 name: Pauline Hans
+img: PaulineHans.jpg
```

Comment:
You forgot to change the img name in your md profile

Reply:
upsi